client/trino-cli/src/main/java/io/trino/cli/Console.java

```diff
@@ -191,7 +189,18 @@ public boolean run()
                         clientOptions.progress.orElse(false));
             }
 
-            runConsole(queryRunner, exiting, clientOptions.editingMode, clientOptions.progress.orElse(true), clientOptions.disableAutoSuggestion);
+            Optional<String> pager = clientOptions.pager;
+            if (!pager.isPresent()) {
```

Comment:
isEmpty()

Reply:
The CLI is built for Java 8 and `isEmpty` is not available.